Composer/packages/server/src/locales/en-US.json

```diff
@@ -299,6 +299,9 @@
   "apr_9_2020_3c8b47d7": {
     "message": "Apr 9, 2020"
   },
+  "are_you_sure_you_want_to_delete_your_bot_this_acti_214a9e11": {
+    "message": "Are you sure you want to delete your bot? This action cannot be undone and your bot and all related files in the bot project folder will be permanently deleted. Your Azure resources will remain unchanged."
```

Comment:
Super nit:  I would have 4 sentences here:
Are you sure you want to delete your bot? This action cannot be undone. Your bot and all related files in the bot project folder will be permanently deleted. Your Azure resources will remain unchanged.

Reply:
@GeoffCoxMSFT Out of curiosity, why would you opt for separate sentences?